### PR TITLE
Provide path to binary instead of install dir

### PR DESCRIPTION
**Pull request by jfacorro**
@robertoaloi

### CODE REVIEW

File: lsp-erlang.el [hunk 1]
```diff
@@ -31,15 +31,15 @@
   :group 'lsp-mode
   :link '(url-link "https://github.com/erlang-ls/erlang_ls"))
 
-(defcustom lsp-erlang-server-install-dir
-  "."
-  "Path to the Erlang Language Server installation dir."
+(defcustom lsp-erlang-server-path
```
Comment (robertoaloi) [inline lsp-erlang.el]: I like the change, but please update the README at https://github.com/erlang-ls/erlang_ls/ to reflect the new variable.

Comment (robertoaloi) [reply lsp-erlang.el]: Also, thanks for the explicit ping on the PR. Is there any way to be automatically pinged if this file is modified?

Comment (yyoncho) [reply lsp-erlang.el]: I guess you may subscribe to all notifications and filter only the notifications containing the filename.

Comment (robertoaloi) [reply lsp-erlang.el]: @yyoncho This sounds like a more powerful alternative: https://help.github.com/en/articles/about-code-owners

Comment (robertoaloi) [reply lsp-erlang.el]: @jfacorro I updated the README: https://github.com/erlang-ls/erlang_ls/commit/6477ec23b75e992f2dec5db39c8898be0bcc38d7

**Pull request merged**